src/extension/vscode.proposed.chatReferenceDiagnostic.d.ts

```diff
@@ -18,6 +18,6 @@ declare module 'vscode' {
 		 */
 		readonly diagnostics: [Uri, Diagnostic[]][];
 
-		protected constructor(diagnostics: [Uri, Diagnostic[]][]);
+		constructor(diagnostics: [Uri, Diagnostic[]][]);
```

Comment:
Sneaked another change related to https://github.com/microsoft/vscode/pull/279729